CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+
+#### Approvers
+The PR author `/assign`s the suggested approvers to approve the PR.
+Only the approvers listed in the OWNERS file can approve the PR.
+Approvers look for holistic acceptance criteria, including dependencies with other features, forwards/backwards compatibility, API and flag definitions, etc
+If the changes made by the PR look good to them, a reviewer types `/approve` in a PR comment; if they change their mind, `/approve cancel`.
+
+### PR Merge Automation
+Once all the conditions are satisfied, [Tide](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/README.md) merges the PR.
+Sanity Check After Merging
+The PR author is responsible for ensuring the change lands in the production system and works as expected. If the users get impacted up to an error caused by the change, revert the PR and roll back the system to give space to think of the fix. If no one is around to approve the reverted PR, impersonate the merge robot whose credentials are stored in BitWarden to do the green button merge with a comment on the PR indicating who is behind the scene. This is only for the cases where our production system does not work properly and the revert is going to fix it.
+
+## The Code Review Guidelines
+In general, reviewers should favor approving a PR once it is in a state where it definitely improves the overall code health of the system being worked on, even if it isn’t perfect.
+
+### Design and Functionality
+Reviewers should take the followings into account:
+- Does the PR bring a useful feature to the system? Does the PR implement the feature requested in a Jira card from the DP team’s current sprint?
+- Should the functionality be refactored into an existing tool? For example, if someone added a new tool rather than enhancing an existing one.
+- Is there any existing code that can be refactored and/or reused in the PR?
+How to verify if the new feature or the fix from the PR works after it lands in production?
+
+### PR size
+
+If the size of a PR is too big, reviewers can think about breaking it into smaller ones.
+
+### Tests
+[The Pull Request Workflow section](https://docs.google.com/document/d/1Qd4qcRHUxk5-eiFIjQm2TTH1TaGQ-zhbphLNXxyvr00/edit?usp=sharing) as described in “Definition of Done”
+
+### Naming
+A good name is long enough to fully communicate what the item is or does, without being so long that it becomes hard to read.
+
+### Comments
+- Is the comment useful? E.g., explaining why some code exists.
+- Is there a TODO comment that can be removed since  the PR does the TODO?
+- GoLang documentation on Classes/Functions/Fields are also comments. Are they written properly?
+
+### Documentation
+Should the change from the PR be documented in the README file? Should the README file be created for the PR if it is for a new tool? Should the ci-docs site be updated accordingly in case that the change will impact our CI users?
```

Comment:
```suggestion
Should the change from the PR be documented in the README file? Should the README file be created for the PR if it is for a new tool? Should the ci-docs site be updated accordingly in case the change will impact our CI users?
```

Reply:
Updated.